apps/site/src/content/blog/state-of-the-budgie-2023.mdx

```diff
@@ -0,0 +1,203 @@
+#### Power Dialog
+
+![](/images/blog/2023/01/BudgiePowerDialog.jpg)
+
+A new Power Dialog for common actions such as shutdown, reboot, suspend, and more was introduced. This functionality replaced the actions exposed through the User Indicator applet, no longer requiring it to be added to the panel.
+
+This new dialog enables third-party applet developers, such as those making their own menu implementations, to easily incorporate our power dialog options without much effort or requiring them to implement duplicate functionality. This was achieved by making the menu D-Bus invokable, and keyboard driver users can invoke the power dialog with our handy (Win/Super)+Shift+P keybind.
+
+#### Other
+
+Here are some other smaller improvements sprinkled across the year:
+
+1. IconTasklist scaling for icons was introduced, enabling dynamic scaling for smaller and larger panel sizes alike while retaining desirable icon sizes for panels in "Dock mode".
+2. The previously third-party Budgie Trash Applet, developed by Buddies of Budgie team member Evan Maddock (EbonJaeger), was merged into mainline Budgie in 10.8.
+3. Privilege Escalation Dialogs show the action description and ID so you know what is requesting access.
+4. power-profiles-daemon support was integrated into our battery indicator, so you can switch between Balanced, Power Saver, and Performance modes depending on device support.
+
+#### Architectural Improvements
+
+##### Application Indexer
+
+Budgie 10.7 introduced a new application indexer, replacing our use of libgnome-menus and unifying our logic used in both Budgie Menu and Budgie Run Dialog, for consistency in the applications we show. This indexer is more reliable across a wider variety of directories which are used to store desktop files.
+
+##### Status Notifier Support
+
+![](/images/blog/2023/08/budgie-10.8/system-tray.jpg)
+
+The System Tray applet was completely overhauled in Budgie 10.8. Historically we have used either `na-tray` or the XEmbed Protocol for the creation, management, and rendering of trays icons. In Budgie 10.8, our System Tray adopted the [Status Notifier](https://www.freedesktop.org/wiki/Specifications/StatusNotifierItem/) specification. This is a modern, vendor-agnostic solution that enables applications to communicate all relevant information for the rendering of their tray icon over DBus rather than being responsible for drawing tray icons and menus themselves, promoting the development of catered solutions by desktop environments and those developing panels for user experiences such as tiling window managers.
+
+The result of this on the Budgie Desktop side is that we can draw all menus and items in a manner that is visually consistent. Going into Budgie 11, we will have a similar solution and paired with multiple panels across multiple monitors (a requirement for Budgie 11), you will even be able to have multiple tray applet instances enabled, which was not otherwise possible with the XEmbed-based solutions.
```

Comment:
```suggestion
The result of this on the Budgie Desktop side is that we can draw all menus and items in a manner that is visually consistent. Going into Budgie 11, we will have a similar solution and paired with multiple panels across multiple monitors (a requirement for Budgie 11); you will even be able to have multiple tray applet instances enabled, which was not otherwise possible with the XEmbed-based solutions.
```